spinalcordtoolbox/scripts/sct_fmri_compute_tsnr.py

```diff
@@ -45,11 +51,28 @@ def compute(self):
         data_std = np.std(data, 3, ddof=1)
         # compute TSNR
         data_tsnr = data_mean / data_std
+        # Change nan to zeros
+        data_tsnr[np.isnan(data_tsnr)] = 0
+        # compute mean tSNR per slice if mask is there
+        if self.mask is not None:
+            mask = Image(self.mask)
+            orientation_mask = mask.orientation
+            if not orientation_mask == 'RPI':
+                mask = mask.change_orientation('RPI')
+            data_tsnr_masked = data_tsnr * mask.data
+            for z in range(data_tsnr_masked.shape[-1]):
+                # Display result
+                tsnr_roi = (data_tsnr_masked[:, :, z])[data_tsnr_masked[:, :, z] != 0].mean()
+                printv(f'\nSlice {z},  tSNR = {tsnr_roi:.2f}', type='info')  # TODO: display slice number in original space
```

Comment:
@joshuacwnewton maybe we should address this todo before merging?

Reply:
Yes, definitely! It should be fairly easy to do, too, since we have that new function written by @valosekj that converts RPI slice numbers back to original slice numbers.

Reply:
@valosekj would you mind pointing me to that function?

Reply:
done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4602/commits/8a248040776a169db9b653b5ec81301a4dbbfe36